base/applications/taskmgr/lang/en-US.rc

```diff
@@ -135,6 +135,9 @@ BEGIN
             MENUITEM "&Low", ID_PROCESS_PAGE_SETPRIORITY_LOW
         END
         MENUITEM "Set &Affinity...", ID_PROCESS_PAGE_SETAFFINITY
+        MENUITEM SEPARATOR
+        MENUITEM "&Properties", ID_PROCESS_PAGE_PROPERTIES
+        MENUITEM "&Open File Location", ID_PROCESS_PAGE_OPENFILELOCATION
```

Comment:
This needs to be done/duplicated for all other languages too ofc.

Reply:
OK.